### DISCUSSION TITLE
webui stable diffusion FORGE NEW UPDATE - Deforum Not working

### DISCUSSION THREAD

**Discussion by ibrahim888k** [community/community#137518] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello ,
i've noticed that the extension deforum is not working on the new update of the forge .. is there any updates on the extension being worked on ? how to go back to the older version  ? Any information about this topic would be great

**Comment by LiteBrite82** [comment]
Hi @ibrahim888k ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [stable-diffusion-webui repository](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
